backend/routes/links.js

```diff
@@ -0,0 +1,43 @@
+const express = require("express");
+const router = express.Router();
+const db = require("../config/database");
+
+const Link = db.Link;
+
+router.get("/", (req, res, next) => {
+    Link.findAll({
+        limit: 2, 
+        order: '"id" ASC'
+    }).then(data=> res.send(data));
+});
+router.get("/:id", (req, res, next) => {
+    Link .findAll({ where: {
```

Comment:
Shouldn't this `Link.findAll` be a `.findOne`?

Reply:
fixed!